examples/publication/frontend/best-practice-guidance.json

```diff
@@ -190,6 +190,21 @@
         "links": {
         }
       }
+    ],
+    "secondary_to_step_by_steps": [
```

Comment:
This should have the full step by step content item in it, e.g. https://github.com/alphagov/govuk-content-schemas/blob/master/examples/guide/frontend/guide-with-step-navs.json#L18

Reply:
👍 Added that in now. I'm looking at publishing-api